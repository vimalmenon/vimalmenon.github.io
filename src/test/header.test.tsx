import { render } from '@testing-library/react';
import { Header } from '@common';

describe('Header component', () => {
  it('Snapshot Testing', () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
