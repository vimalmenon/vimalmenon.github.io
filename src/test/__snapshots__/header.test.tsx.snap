// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Snapshot Testing 1`] = `
<div>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-d8mtz7-MuiPaper-root-MuiAppBar-root"
    style="--Paper-shadow: 0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12);"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthXl css-hhdjsd-MuiContainer-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-dense css-nk5xtv-MuiToolbar-root"
      >
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-w3fons-MuiTypography-root-MuiLink-root"
          href="/"
        >
          Vimal Menon
        </a>
        <div
          class="MuiBox-root css-1i27l4i"
        >
          <div
            class="MuiBox-root css-suo9w"
          >
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-180oxxk-MuiTypography-root-MuiLink-root"
              href="/"
            >
              Home
            </a>
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-180oxxk-MuiTypography-root-MuiLink-root"
              href="/blogs"
            >
              Blogs
            </a>
          </div>
          <div
            class="MuiBox-root css-0"
          >
            <button
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
              tabindex="0"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                data-testid="Brightness4Icon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 8.69V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12zM12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6c3.31 0 6 2.69 6 6s-2.69 6-6 6"
                />
              </svg>
            </button>
          </div>
        </div>
      </div>
      <div
        class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-dense css-1vill88-MuiToolbar-root"
      >
        <button
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
            data-testid="MenuIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3zm0-5h18v-2H3zm0-7v2h18V6z"
            />
          </svg>
        </button>
      </div>
    </div>
  </header>
</div>
`;
